qchem/pennylane_qchem/qchem.py

```diff
@@ -531,7 +531,7 @@ def generate_mol_hamiltonian(
                 map the second-quantized electronic Hamiltonian to the qubit Hamiltonian
         outpath (str): path to the directory containing output files
     Returns:
-        tuple(QubitOperator, int): the fermionic-to-qubit transformed Hamiltonian
+        tuple(pennylane.beta.vqe.Hamiltonian, int): the fermionic-to-qubit transformed Hamiltonian
```

Comment:
maybe above:
tuple(pennylane.beta.vqe.Hamiltonian, int): the fermionic Hamiltonian represented as linear combination of Pauli operators measurable in PennyLane